### ISSUE TITLE
how to increase sidebar icon size?

### ISSUE THREAD

**Issue by betorobson** [ihodev/sublime-da-ui#26]

**Comment by ihodev** [comment]
Do you mean the size of the file icons? Please, provide more details :wink:

**Comment by betorobson** [comment]
Yes, I mean the icon size, actually it is possible increase the font size, but I could not find the icon size.
For the font I'm using the property: "sidebar_label_font_size"

**Comment by betorobson** [comment]
![2017-10-17_1011_001](https://user-images.githubusercontent.com/13212208/31664329-c4e13a8c-b323-11e7-83d3-bbd8a81ccfad.png)
hummm, I just found a setting on A File Icon, change size from 8 to 14 wont make the right effect. Take a look on these two screenshots, first is size 8 and them size 14.
![2017-10-17_1011](https://user-images.githubusercontent.com/13212208/31664325-c2591bd6-b323-11e7-9242-0e777eb4b03a.png)

**Comment by ihodev** [comment]
Ok, I understood. Yes, the size of the icon should be adjusted in `A File Icon` settings, but there is some bug, which will be fixed in the next version of the `A File Icon`.

Currently, you can open `Packages/zzz A File Icon zzz/patches/general/multi/DA.sublime-theme` file and
add next code to the first `"icon_file_type"` rule:

```js
    "layer0.inner_margin": 0,
    "layer0.draw_center": true,
```

However I have plans to deprecate the changing of the icon size, because in any case they are blurry. As for the aligning, it will be fixed.

**Comment by ihodev** [comment]
Blocked by https://github.com/ihodev/a-file-icon/issues/139.

**Comment by betorobson** [comment]
Before change the issues title, I would like to ask you if you agree changing it for alignment of the file `Fix file icons alignment based on font size`

**Comment by borela** [comment]
@ihodev 
> However I have plans to deprecate the changing of the icon size, because in any case they are blurry
Will it be possible to still use bigger icons? Having options like small, normal, big, bigger, etc...
